README.markdown

```diff
@@ -1048,7 +1048,7 @@ let message = "You cannot charge the flux " +
 
 ## Organization and Bundle Identifier
 
-Where an Xcode project is involved, the organization should be set to `Ray Wenderlich` and the Bundle Identifier set to `com.raywenderlich.TutorialName` where `TutorialName` is the name of the tutorial project.
+Where an Xcode project is involved, the organization should be set to `Deputec Pty Ltd` and the Bundle Identifier set to `com.deputy.ProjectName` where `ProjectName` is the name of the project.
```

Comment:
Mine sets `Deputy` - does it matter which we use? Could be `Deputy` or `Deputec Pty Ltd`. Same thing?

Reply:
Also, our bundle ID is `com.deputy.prod` - `prod` is not the name of the project. Perhaps just a rule that it should have a _prefix_ of `com.deputy.`